src/fields/field.impl.hh

```diff
@@ -398,8 +398,9 @@ void Field<spacedim, Value>::observe_output(std::shared_ptr<Observe> observe)
 
     if (observe->point_ds()->size() == 0) return;
 
```

Comment:
observe_output would be necessary only for DarcyMH, so we should remove DarcyMH first (in master) and remove this method as well.

Reply:
observe_output is not used now, method will be removed, assembly classes used in equations and AssemblyObserveOutput class are independent.

Reply:
observe_output is not used now, method will be removed, assembly classes used in equations and AssemblyObserveOutput class are independent.